data/upstream.yaml

```diff
@@ -150,6 +150,10 @@ fail2ban:
       bug: https://bugzilla.redhat.com/show_bug.cgi?id=1282498
   note: |
       Upstream claims support of python-3.2+
+fife:
+    status: in-progress
+    links:
+        bug: https://github.com/fifengine/fifengine/issues/631
```

Comment:
I don't think this is in progress – there's an issue reported, but no one seems to be working on this.